### PR TITLE
patch: initialization of textStyle object

### PR DESCRIPTION
**Pull request by dvh91**
### Description of the Changes

textStyle object initialized so the UI could understand what the initial textStyle is and show accordingly the captions style options.

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment 
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/player.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ export default class Player extends FakeEventTarget {
    * @type {TextStyle}
    * @private
    */
-  _textStyle: TextStyle;
+  _textStyle: TextStyle = new TextStyle();
```
Comment (dan-ziv) [inline src/player.js]: Move initialization to constructor

**Comment by OrenMe** [comment]
@dvh91 don't know if you deleted the default Pull Request template or it didn't appear but we have it there for a reason so please add it to the first comment of this PR(just edit it) and add details and tick the corresponding check boxes after you save it.
Thanks!

**Comment by OrenMe** [comment]
@dan-ziv ES6 classes member initialization can be done like this in general, and it is exactly the same as constructor initialization AFAIK.
Only issue is the fact we do some in constructor and some in member definition, so there's no alignment.
By the way, if you check the actual transpiled class you will see it does the _textTrack init in the class function definition, e.g.
```javascript
function Player() {
   ...
   this._textTrack = new ...
   ...
}
```

**Comment by dvh91** [comment]
@OrenMe - Updated the first comment with the pull request template.

**Comment by dan-ziv** [comment]
@OrenMe 
I know that,
The reason I asked this is only for consistency.

**Pull request merged**